### PR TITLE
fix(config): updates for Illumino Dimmer Switch firmware 1.07 and Illumino Switch firmware 1.05

### LINKED ISSUE #4573: update Update illumino Dimmer Switch config to reflect V1.07 firmware
new firmware was released that add additional options

https://help.aeotec.com/support/solutions/articles/6000257235-update-illumino-dimmer-switch-v1-07

### PR DESCRIPTION
**Pull request by lukasz-karolewski**
updating config to reflect new option in Illumino Dimmer Switch firmware 1.07 and Illumino Switch firmware 1.05

https://help.aeotec.com/support/solutions/articles/6000257235-update-illumino-dimmer-switch-v1-07
https://help.aeotec.com/support/solutions/articles/6000257236-update-illumino-switch-v1-05

fixes #4573

### CODE REVIEW

File: packages/config/config/devices/0x0086/templates/aeotec_template.json [hunk 1]
```diff
@@ -3298,5 +3306,21 @@
...
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Enabled",
+				"value": 1
+			}
+		]
 	}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0086/templates/aeotec_template.json]: This template already exists

File: packages/config/config/devices/0x0086/templates/aeotec_template.json [hunk 2]
```diff
@@ -1834,6 +1834,14 @@
...
 				"value": 2
+			},
+			{
+				"label": "Inverted LED state based on output",
+				"value": 3
+			},
+			{
+				"label": "LED always on, blinks when state changed",
+				"value": 3
 			}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0086/templates/aeotec_template.json]: First, value 3 is duplicated

Second, don't edit templates which are used in multiple devices to include a change that is only correct for some firmware versions of a single device. Instead add a new template with the new options and conditionally use it depending on the firmware version. Example [here](https://zwave-js.github.io/node-zwave-js/#/config-files/conditional-settings) at the bottom.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0086/templates/aeotec_template.json (hunk 1 above)
Comment (lukasz-karolewski) [reply packages/config/config/devices/0x0086/templates/aeotec_template.json]: switched to master_template.json#base_enable_disable now, thanks

File: packages/config/config/devices/0x0086/templates/aeotec_template.json (hunk 2 above)
Comment (lukasz-karolewski) [reply packages/config/config/devices/0x0086/templates/aeotec_template.json]: thank you for pointing me in the right direction,

Both devices got new firmware with same new options, adjusting to reflect that.

**Comment by lukasz-karolewski** [comment]
I should have opened as a draft PR, still trying to figure out how to best test the new config locally

File: packages/config/config/devices/0x0371/zwa037.json [hunk 3]
```diff
@@ -139,6 +144,12 @@
 		{
 			"#": "133",
 			"$import": "~/0x0086/templates/aeotec_template.json#custom_brightness_99"
+		},
+		{
+			"#": "252",
+			"$if": "firmwareVersion >= 1.7",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Manual button press and parameter setting lock"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa037.json]:
```suggestion
			"label": "Lock Configuration and Manual Button Press"
```

File: packages/config/config/devices/0x0371/zwa038.json [hunk 4]
```diff
@@ -93,6 +98,12 @@
 		{
 			"#": "122",
 			"$import": "~/templates/master_template.json#orientation_momentary"
+		},
+		{
+			"#": "252",
+			"$if": "firmwareVersion >= 1.5",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Manual button press and parameter setting lock"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa038.json]:
```suggestion
			"label": "Lock Configuration and Manual Button Press"
```

File: packages/config/config/devices/0x0371/zwa038.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
 	"manufacturer": "Aeotec Ltd.",
 	"manufacturerId": "0x0371",
 	"label": "ZWA038",
-	"description": "illumino Wall Switch",
+	"description": "Illumino Wall Switch",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa038.json]: AFAIK they are called "illumino" (lowercase)
https://aeotec.com/z-wave-wall-switch/index.html

```suggestion
	"description": "illumino Wall Switch",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
Just a few nits. I'm also going to ask @blhoward2 to comment on the new option labels, since we want to make sure to have them as consistent as possible.

**Review by blhoward2 — APPROVED** [review]
I'm ok with these option labels as they now read. They duplicate what we have in the template where we already have the same (e.g. "LED follows switch state").

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**